api/validation/drink.js

```diff
@@ -10,7 +10,8 @@ const drinkSchema = Joi.object().keys({
     .required(),
   tastingNotes: Joi.string().required(),
   comments: Joi.string().allow(""),
-  image: Joi.string().required(),
+  // image: Joi.string().required(),
+  image: Joi.string().allow(""),
```

Comment:
Do we want to allow users to submit empty strings for the image?

Reply:
ugh. I had that part disabled for testing. I set it as required in the model, but not back to required in Joi.